CONTRIBUTING.md

```diff
@@ -46,6 +48,99 @@ In case you need another package than the ones installed (see the [DESCRIPTION f
 but please consider this carefully and ask one of the admins if in doubt.
 Note also that any dependency packages needed by the packages listed in the [DESCRIPTION file](DESCRIPTION) are also available in the project and they are listed in [the lock file](renv.lock).
 
+## `(R)markdown` syntax and learning `Rmarkdown`
+
+This repository contains the source documents of protocols.
+These source documents are written in `Rmarkdown` language, which is a combination of a simple markup language and R code written in R chunks.
+
+RStudio has both a `source` mode and a [`visual` mode](https://rstudio.github.io/visual-markdown-editing/) to work with these kind of documents.
+The `source` mode is a plain text file, whereas the `visual` mode is a WYSIWYG editor mode which will be more familiar to Microsoft-Word users.
+The RStudio project associated with the `protocolsource` repository is configured to use [`canonical visual mode markdown`](https://rstudio.github.io/visual-markdown-editing/markdown.html#canonical-mode) in source mode at project level.
+This setting ensures that the same markdown syntax is written to disc no matter if you edited the document in `source` mode or in `visual` mode.
+For instance, you could write an unordered list in markdow as follows in source mode:
+
+```
+* item 1
+* item 2
+```
+
+When you press save - due to the canonical mode setting - this will be automatically changed to:
+
+```
+-   item 1
+-   item 2
+```
+
+Similarly, html comments that span multiple lines will be wrapped in a html block:
```

Comment:
Why are these called html comments?  I thought they were just regular markdown comments (but I'm not an expert in markup languages, and my html knowledge hides somewhere far in my memory...).

Reply:
It's html: see https://www.w3schools.com/tags/tag_comment.asp